src/iperf_api.c

```diff
@@ -4498,6 +4498,7 @@ iperf_new_stream(struct iperf_test *test, int s, int sender)
 
     memset(sp, 0, sizeof(struct iperf_stream));
 
+    sp->thread_created = 0;
```

Comment:
Technically don't need this line because of the `memset()` above. It's not wrong to have it here though.

Reply:
Line removed.